css-canvas-medical/styles.css

```diff
@@ -0,0 +1,248 @@
+/* nav bar */
+.navbar-text {
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  color: rgb(208 208 208);
+  word-spacing: 10px;
+  font-size: 13px;
+  margin-right: 50px;
+}
+
+#navbar-line {
+  width: 1200px;
+  border-color: rgb(239 239 242);
+  border-style: solid;
+}
+
+#container {
+  max-width: 100%;
+  height: 1980px;
+}
+
+#hero-image {
+  width: 1000px;
+  height: 500px;
+  margin: auto;
+  position: relative;
+  display: block;
+  filter: brightness(80%);
+  filter: contrast(60%);
+}
+
+#hero-header {
+  position: absolute;
+  top: 180px;
+  left: 300px;
+  width: 400px;
+  font-family: "Tiempos Headline", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  color: white;
+  font-size: 35px;
+}
+
+#hero-text {
+  position: absolute;
+  width: 500px;
+  top: 320px;
+  left: 300px;
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  color: white;
+  font-size: 15px;
+  line-height: 27px;
+}
+
+#header-button {
+  background-color: rgb(41 68 100);
+  color: white;
+  position: absolute;
+  top: 540px;
+  left: 300px;
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  padding: 10px 30px;
+  border-style: solid;
+  border-color: rgb(41 68 100);
+  border-radius: 5px;
+  font-size: large;
+}
+
+.row {
+  display: flex;
+  margin: auto;
+}
+
+.column {
+  margin: auto;
+}
```

Comment:
`column` classes are meant to have either `flex-basis` or `width`, and that's it. This is not a proper layout class setup, I suggest reviewing `css-layout-classes` for a refresher.